context/app/routes_browse.py

```diff
@@ -50,15 +50,20 @@ def details(type, uuid):
     if type != actual_type:
         return redirect(url_for('routes_browse.details', type=actual_type, uuid=uuid))
 
-    template = 'pages/base_react.html'
-    conf_cells_and_lifted_uuid = client.get_vitessce_conf_cells_and_lifted_uuid(entity)
     flask_data = {
         **get_default_flask_data(),
         'entity': entity,
-        'vitessce_conf': conf_cells_and_lifted_uuid.vitessce_conf.conf,
-        'has_notebook': conf_cells_and_lifted_uuid.vitessce_conf.cells is not None,
-        'vis_lifted_uuid': conf_cells_and_lifted_uuid.vis_lifted_uuid
     }
+
+    if type == 'dataset':
```

Comment:
Should we use enums for these values at some point or are we fine with using 'dataset' and its variants as needed?

Reply:
I don't really care strongly... perhaps not an enum, but just a module that defines names. Will file.